src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage_scenarios_non_ascii_chars.py

```diff
@@ -0,0 +1,64 @@
+# This Python file uses the following encoding: utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+from azure.cli.testsdk import (LiveScenarioTest, ResourceGroupPreparer, StorageAccountPreparer, JMESPathCheck)
+from .storage_test_util import StorageScenarioMixin
+
+
+class StorageCopyTests(StorageScenarioMixin, LiveScenarioTest):
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer()
+    def test_storage_blob_copy_non_ascii(self, storage_account_info):
+        src_container = self.create_container(storage_account_info)
+        dst_container = self.create_container(storage_account_info)
+
+        source_file = self.create_temp_file(16, full_random=False)
+        blobs = ['blobğşŞ', 'blobÉ®']
```

Comment:
Please open an issue for this shortcoming of VCRpy in https://github.com/azure/azure-python-devtools.